### PR TITLE
Using glyph references

### PR DESCRIPTION
**Pull request by jishnu7**
Glyphs can be optimized more. This PR doesn't cover possible references.

### CODE REVIEW

**Comment by jishnu7** [comment]
@santhoshtr @rajeeshknambiar

File: README.md [hunk 1]
```diff
@@ -10,3 +10,12 @@ The source code repository of malayalam unicode fonts developed or maintained by
 * Rachana
 * Raghu Malayalam Sans
 * Suruma
+
+How to Build
+=====
```
Comment (santhoshtr) [inline README.md]: We have a makefile that does all these. Following targets are already defined: default, clean, compile, install, uninstall, test,  webfonts, sdist

Comment (jishnu7) [inline README.md]: removed.

Comment (rajeeshknambiar) [inline README.md]: Nice!

Has this been tested for regression? i.e, all changed glyphs?

On Sat, Sep 27, 2014 at 6:56 PM, Jishnu Mohan notifications@github.com
wrote:

> In README.md:
> 
> > @@ -10,3 +10,12 @@ The source code repository of malayalam unicode fonts developed or maintained by
> > - Rachana
> > - Raghu Malayalam Sans
> > - Suruma
> >   +
> >   +How to Build
> >   +=====
> 
> removed.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/smc/fonts/pull/8/files#r18124608.

## 

Rajeesh

**Comment by rajeeshknambiar** [comment]
Reviewed the changes, looks good!
Could you update following glyphs in similar fashion?

k1nhu1, k1nhu2
k1shnh, k1shnhu1, k1shnhu2
k1shm1, k1shm1u1, k1shm1u2
k1shm1r1
th1s1th2, th1s1th2u1, th1s1th2u2
th1s1th2r1
th1s1n1, th1s1n1u1, th1s1n1u2
th1s1m1, th1s1m1u1, th1s1m1u2, th1s1m1r1
th4n1, th4n1u1, th4n1u2
th4m1, th4m1u1, th4m1u2
th4m1r1
s1k1, s1k1u1, s1k1u2
s1k1r1
s1k1k1, s1k1k1u1, s1k1k1u2
s1k1k1r1
s1k1r3, s1k1r3u1, s1k1r3u2
s1k2, s1k2u1, s1k2u2
s1t1, s1t1u1, s1t1u2
s1t1r3, s1t1r3u1, s1t1r3u2
s1th1, s1th1u1, s1th1u2
s1th1r1
s1th1r2
s1th1r3, s1th1r3u1, s1th1r3u2
s1th2, s1th2u1, s1th2u2
s1th2r1
s1n1, s1n1u1, s1n1u2
s1p1, s1p1u1, s1p1u2
s1p1r1
s1p1r3, s1p1r3u1, s1p1r3u2
s1p2, s1p2u1, s1p2u2
s1m1, s1m1u1, s1m1u2
s1m1r1
s1l3, s1l3u1, s1l3u2
s1s1, s1s1u1, s1s1u2
s1s1r1
s1s1r3, s1s1r3u1, s1s1r3u2
s1rhrh, s1rhrhu1, s1rhrhu2, s1rhrhr3, s1rhrhr3u1, s1rhrhr3u2
zhk1k1, zhk1k1u1, zhk1k1u2
zhn1n1u2

**Comment by jishnu7** [comment]
added k1sh\* and th1s1*.

ക il k1nhu1 and k1nhu2 slightly different than k1. and th4 and ധ in th4\* are slightly different, same for s1\* and zhk*.

Since these are very slight differences, I'm not sure whether these are intentional or not.

**Comment by rajeeshknambiar** [comment]
I couldn't distinguish the difference (don't trust my eyes, though) and most probably such change if exists is unlikely intentional.

**Comment by rajeeshknambiar** [comment]
Jishnu, were those changes made and ready for merge?

**Comment by rajeeshknambiar** [comment]
I have merged the initial pull request and pushed out to savannah. Could you rebase the next changes on top of that?

**Comment by jishnu7** [comment]
Sorry for the late reply. 

I didn't get time to make other changes. I'll do it as a different PR. 

Updated github with latest changes.

**Pull request closed without merging**